src/vimeoplayer/views/Vimeoplayer.js

```diff
@@ -0,0 +1,472 @@
+import $ from 'jquery';
+import _ from 'underscore';
+import Mediaplayer from 'picnic/mediaplayer/views/Mediaplayer';
+import ApiLoader from 'picnic/vimeoplayer/services/ApiLoader';
+
+var DATA_VIDEOID = 'vimeoid',
+	DEFAULTS = {
+		debug: false,
+		loader: new ApiLoader(),
+		trigger: 'a',
+		classLoading: 'loading',
+		classPlaying: 'playing',
+		playerHideSpeed: 300,
+		playerProgressSteps: 5,
+		playerProgressInterval: 1000,
+		playerOptions: {
+			autoplay: true
+		},
+	};
+
```

Comment:
Yes it's cool to have a customizable eventNamespace but this wasn't really my intend. I wanted them to be stored as "constant" beside the `DEFAULTS` var in the style 

```javascript
var
    EVENT_PLAY = 'vimeoplayer:play',
    EVENT_STOP = 'vimeoplayer:stop',
    EVENT_PAUSE = 'vimeoplayer:pause',
    //...
```
 
Maybe we should name the options property `eventNamespace` rather than only `namespace`?